x-pack/platform/plugins/shared/security/server/uiam/uiam_service.ts

```diff
@@ -104,6 +104,8 @@ export class UiamService implements UiamServicePublic {
    */
   async refreshSessionTokens(refreshToken: string) {
     try {
+      this.#logger.debug('Attempting to refresh session tokens with UIAM.');
```

Comment:
nit: this logger is already scoped to UIAM via https://github.com/kc13greiner/kibana/blob/d8fd317c4166c99b3da0e679ca2cc360ec60ea97/x-pack/platform/plugins/shared/security/server/plugin.ts#L412.


```suggestion
      this.#logger.debug('Attempting to refresh session tokens.');
```